Src/CSharpier.CLI/CommandLineFormatter.cs

```diff
@@ -9,9 +9,9 @@
 using CSharpier.Utilities;
 using Microsoft.Extensions.Logging;
 
-namespace CSharpier
+namespace CSharpier.CLI
```

Comment:
Okay, seeing the namespace makes me feel a little more strongly about the casing.

Reply:
Yeah, I think Cli does look better after making the change